### PR TITLE
TW-997: Apply new staring page

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket
- #997 

### Need merge

- https://github.com/linagora/linagora-design-flutter/pull/35

### Resolved

https://github.com/linagora/twake-on-matrix/assets/99852347/9a72a484-39e4-4c35-b76b-907629a00d3e

### CODE REVIEW

File: lib/config/go_routes/go_router.dart [hunk 1]
```diff
@@ -343,6 +364,22 @@ abstract class AppRoutes {
...
+                  redirect: loggedInRedirect,
+                ),
+                GoRoute(
+                  path: 'homeserverpicker',
+                  pageBuilder: (context, state) => defaultPageBuilder(
+                    context,
+                    const HomeserverPicker(),
+                  ),
+                  redirect: loggedInRedirect,
+                ),
```
Comment (imGok) [inline lib/config/go_routes/go_router.dart:382]: why do we need them twice ?

File: lib/pages/homeserver_picker/homeserver_picker_view.dart [hunk 2]
```diff
@@ -120,12 +115,12 @@ class HomeserverPickerView extends StatelessWidget {
         );
     switch (controller.state) {
       case HomeserverState.otherLoginMethod:
-        return Text(L10n.of(context)!.loginWithSSO, style: textStyle);
+        return Text(L10n.of(context)!.continueProcess, style: textStyle);
       case HomeserverState.loading:
         return const Expanded(child: LinearProgressIndicator());
       case HomeserverState.ssoLoginServer:
       case HomeserverState.wrongServerName:
-        return Text(L10n.of(context)!.loginWithSSO, style: textStyle);
```
Comment (imGok) [inline lib/pages/homeserver_picker/homeserver_picker_view.dart:128]: delete it from intl files if it's not used anymore imo

File: lib/config/go_routes/go_router.dart (hunk 1 above)
Comment (nqhhdev) [reply lib/config/go_routes/go_router.dart:382]: Apply for new design

File: lib/pages/twake_welcome/twake_welcome.dart [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+class TwakeWelcome extends StatefulWidget {
+  const TwakeWelcome({super.key});
+
+  @override
+  State<TwakeWelcome> createState() => TwakeWelcomeController();
+}
+
+class TwakeWelcomeController extends State<TwakeWelcome> {
+  void goToLinagoraHomeserver() {
```
Comment (hoangdat) [inline lib/pages/twake_welcome/twake_welcome.dart:13]:
```suggestion
  void goToTwakeIdScreen() {
```

**Review by hoangdat — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**